### PR TITLE
refactor: components using old theming

### PR DESCRIPTION
**Pull request by wackerow**
## Description
- QuizRadioGroup buttons
- SendFromContacts
- ReceiveEther
- ReceivedEther

Updated to tailwind; colors updated to use latest theming

## Related Issue
Theming updates

### CODE REVIEW

File: src/components/Quiz/QuizWidget/QuizRadioGroup.tsx [hunk 1]
```diff
@@ -4,7 +4,6 @@ import {
   Box,
   chakra,
   ChakraProps,
-  Circle,
   HStack,
   Stack,
   Text,
```
Comment (pettinarip) [inline src/components/Quiz/QuizWidget/QuizRadioGroup.tsx:9]: Is it possible to migrate this component completely to Tw? I feel that we only need to change a few components and avoid leaving this in a hybrid situation.

**Review by pettinarip — CHANGES_REQUESTED** [review]
Looks good @wackerow thanks for the migration of these components. Left a small request.

File: src/components/Quiz/QuizWidget/QuizRadioGroup.tsx (hunk 1 above)
Comment (wackerow) [reply src/components/Quiz/QuizWidget/QuizRadioGroup.tsx:9]: @pettinarip I started that and it led to a few complications that I didn't anticipate. For the sake of keeping scope of this PR trimmed to just the color updates, mind if I branch off and patch this up separately?

Comment (pettinarip) [reply src/components/Quiz/QuizWidget/QuizRadioGroup.tsx:9]: Got it. Sure, we can tackle that separately.

**Review by pettinarip — APPROVED** [review]

**Pull request merged**